robot_manager:
  ros__parameters:
    controller_ip: "0.0.0.0"
    control_mode: "position"
    command_mode: "position"
    receive_multiplier: 1
    send_period_ms: 10
    joint_damping: [0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7]
    joint_stiffness: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0]
